modules/actions/split.js

```diff
@@ -313,12 +402,37 @@ export function actionSplit(nodeIds, newWayIds) {
 
 
     action.disabled = function(graph) {
-        for (var i = 0; i < nodeIds.length; i++) {
-            var nodeId = nodeIds[i];
-            var candidates = action.waysForNode(nodeId, graph);
+        for (const nodeId of nodeIds) {
+            const candidates = action.waysForNode(nodeId, graph);
             if (candidates.length === 0 || (_wayIDs && _wayIDs.length !== candidates.length)) {
                 return 'not_eligible';
             }
+            for (const way of candidates) {
+                const parentRelations = graph.parentRelations(way);
+                for (const parentRelation of parentRelations) {
+                    if (parentRelation.hasFromViaTo()) {
+                        // turn restrictions: via memebers must be loaded
+                        const vias = parentRelation.membersByRole('via');
+                        if (!vias.every(via => graph.hasEntity(via.id))) {
+                            return 'parent_incomplete';
+                        }
+                    } else {
+                        // other relations (e.g. route relations): at least one members before or after way must be present
+                        for (let i = 0; i < parentRelation.members.length; i++) {
+                            if (parentRelation.members[i].id === way.id) {
+                                const memberBeforePresent = i > 0 && graph.hasEntity(parentRelation.members[i - 1].id);
+                                const memberAfterPresent = i < parentRelation.members.length - 1 && graph.hasEntity(parentRelation.members[i + 1].id);
+                                if (!memberBeforePresent && !memberAfterPresent) {
+                                    return 'parent_incomplete';
```

Comment:
not to self: maybe add `…&& !parentRelation.members.length > 1` (otherwise it will fail for newly created routes with only 1 member to start with).